src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -103,8 +103,9 @@ public CCDCaseApi(
     }
 
     public List<Claim> getByDefendantId(String id, String authorisation) {
+        LOGGER.debug("ccd search is by authorisation instead of defendant id {}", id);
```

Comment:
@dharmendrak if both getByDefendantId and getByDefendantEmail are going to do the same thing, why have two different method that says the wrong thing... Just merge them to getByDefendantId.... more explicit and clean.